rosbag2_transport/src/rosbag2_transport/player.cpp

```diff
@@ -245,26 +237,21 @@ void Player::play()
         std::chrono::nanoseconds duration(delay.nanoseconds());
         std::this_thread::sleep_for(duration);
       }
-      rcutils_time_point_value_t starting_time = 0;
+      // no need to open and close reader all the time
+      // just seek to the right spot
```

Comment:
```suggestion
```

This comment seems to serve to justify the diff, and only make sense with the historical context that we _were_ opening and closing all the time. I think the code makes sense without the comment